app/src/main/java/org/mifos/selfserviceapp/api/ApiEndPoints.java

```diff
@@ -12,4 +12,5 @@
     public static final String AUTHENTICATION = "authentication";
     public static final String CLIENTS = "clients";
     public static final String SAVINGSACOUNTS = "savingsaccounts";
+    public static final String ACCOUNTTRANSFERS = "accounttransfers";
```

Comment:
usually the style here is ACCOUNT_TRANSFERS which makes the name easier to read. without the _ it's a little difficult to understand.